system/animations/BlockFade.tsx

```diff
@@ -0,0 +1,44 @@
+'use client';
+
+import * as Utilities from '@common/utilities';
```

Comment:
Make sure you follow the conventions that we have in other files for the ordering of imports.